packages/web-react/src/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo, createElement, useRef } from 'react';
+import { CollapseProps } from '../../types';
+import { useCollapse } from './useCollapse';
+import { useCollapseStyle } from './useCollapseStyle';
+
+const Collapse = (props: CollapseProps) => {
+  const {
+    id,
+    children,
+    collapsed,
+    responsive,
+    elementType,
+    contentProps,
+    renderTrigger,
+    hideOnCollapse,
+    UNSAFE_className,
```

Comment:
This is part of the rest props and is defined by the `StyleProps` interface which you should extend.